tests/test_emailFinder.py

```diff
@@ -0,0 +1,11 @@
+import sys
+import requests
+
+
+
+
+
+			
+
+def test_emailFinder():
+	assert 1==1
```

Comment:
Work in progress? :)

Reply:
This is just a placeholder, no plan to continue at the moment :)